shared/src/test/diff/mlscript/Methods.mls

```diff
@@ -0,0 +1,126 @@
+//TODO: - ConstraintSolver.extrude TypeRef polarity
+//      - ConstraintSolver.constrain.annoyingImpl missing cases?
```

Comment:
No... I believe they appeared in ascriptions before they were properly handled. So they could be due to mishandling them, or simply a typo that has since been fixed.

Reply:
Cool, let's remove the comment then :^)

```suggestion
```

Reply:
Addressed in 68be5ec98a994c8360d5c552793a8fac415d8e88